src/utility.ts

```diff
@@ -265,17 +265,7 @@ export class Utility {
 
     public static async getInputDevice(deviceItem: DeviceItem, eventName: string, onlyEdgeDevice: boolean = false, iotHubConnectionString?: string): Promise<DeviceItem> {

```

Comment:
Be careful of changing this method. 'entry: "contextMenu" and entry: "commandPalette"' is used to track whether tigger command from context menu or command palette. This change may lead to regression.

I think you could just use Utility.getFilteredDeviceList dirrecly, no need to change this method.

Reply:
OK, got it. So, `getFilteredDeviceList` should be changed into `public`.